src/dev/bazel/pkg_npm_types.bzl

```diff
@@ -97,6 +99,9 @@ def _pkg_npm_types_impl(ctx):
     mnemonic = "AssembleNpmTypesPackage",
     progress_message = "Assembling npm types package %s" % package_dir.short_path,
     executable = "_packager",
+    env = {
+      "FORCE_COLOR": "1"
```

Comment:
was this used for debugging or did you actually found it useful for errors etc?

Reply:
Without it the "ERROR" label isn't red, so I feel like we should leave it